.github/workflows/pre-commit.yml

```diff
@@ -62,8 +62,11 @@ jobs:
 
     - uses: ./.github/actions/install
       with:
+        python-version: ${{ matrix.python.major_dot_minor }}
         development: true
 
-    - uses: pre-commit/action@v2.0.3
-      with:
-        extra_args: --verbose --all-files
+    - uses: chia-network/actions/activate-venv@main
+
+    - run: |
+        pip install pre-commit
```

Comment:
This shouldn't be needed due to:

https://github.com/Chia-Network/chia-blockchain/blob/6d605ae2119aadcfb9269f0e103d294f87d9d064/setup.py#L45

```suggestion
```